src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeRetrieverServiceTest.java

```diff
@@ -40,15 +41,18 @@ public void setUp() {
     @Test
     public void should_return_all_envelopes_successfully_for_a_given_jurisdiction() throws Exception {
         List<Envelope> envelopes = EnvelopeCreator.envelopes();
+        List<EnvelopeResponse> envelopeResponses = EnvelopeCreator.envelopeResponses();
```

Comment:
Why this `EnvelopeCreator.envelopeResponses()` need to return list rather than a single response object ?

Reply:
Just for consistency with the envelopes() method as I like them to be interchangeable moving from the internal model to the external one.